xcp/bootloader.py

```diff
@@ -525,7 +526,8 @@ def writeGrub(self, dst_file = None):
             m = self.menu[label]
             print("\ntitle " + m.title, file=fh)
             if m.hypervisor:
-                print("   kernel " + m.hypervisor + " " + m.hypervisor_args, file=fh)
+                print("   kernel " + m.hypervisor + " " + m.hypervisor_args,
```

Comment:
Didn't you allow up to 100 chars per line, previously?

Reply:
many commits were written before we decided to switch to 100, sure some line wraps are not necessary any more now

Reply:
unwrapped